src/cpp/rediscluster.cpp

```diff
@@ -470,18 +475,22 @@ inline void RedisCluster::_connect(std::string address_port)
         try {
             this->_redis_cluster =
                 new sw::redis::RedisCluster(address_port);
-        }
-        catch (std::bad_alloc& e) {
-            throw std::runtime_error(e.what());
+            return;
         }
         catch (std::exception& e) {
-            delete this->_redis_cluster;
-            this->_redis_cluster = 0;
```

Comment:
Leave in assignment to NULL (and switch 0 => NULL)

Reply:
Agreed.